src/index.html

```diff
@@ -1,13 +1,49 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img class="product-card__image"
+        src="./images/imac.jpeg"
+        alt="Моноблок APPLE A1419"
+      >
+      <a class="product-card__title"
+        href="#"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
```

Comment:
```suggestion
      <a 
        class="product-card__title"
        href="#"
        target="_blank"
        rel="noopener noreferrer"
      >
```
Тут вместо ссылки можно использовать тег заголовка, чтобы выделить важный текст на карточке

Reply:
Я використав а таг, тому що вирішив, що в подальшому це може бути посилання на сторінку з інфо про конкретний товар. Так це переважно є в більшості інтернет-магазинівю